src/st/class-wpml-page-builders-app.php

```diff
@@ -43,7 +43,15 @@ public function load_integration() {
 		if ( $factories ) {
 			foreach ( $factories as $factory ) {
 				$integration = $factory->create();
-				$integration->add_hooks();
+
+				if ( is_array( $integration ) ) {
+					foreach ( $integration as $hooks ) {
```

Comment:
@strategio can you give `$hook` a better name? Maybe `$integration_item` or something close to what it is supposed to be.